dbt/models/iasworld/schema/iasworld.htpar.yml

```diff
@@ -275,17 +217,27 @@ sources:
           - name: subkey
             description: Unique number to allow multiple hearings per level
           - name: taxyr
-            description: '{{ doc("column_taxyr") }}'
+            description: '{{ doc("shared_column_year") }}'
           - name: trans_id
             description: '{{ doc("column_trans_id") }}'
           - name: txpyval
             description: Taxpayer value
           - name: upd_status
             description: '{{ doc("column_upd_status") }}'
+          - name: user38
+            description: '{{ doc("shared_column_appeal_type") }}'
+          - name: user89
+            description: '{{ doc("shared_column_reason_code") }}'
+          - name: user100
+            description: '{{ doc("shared_column_reason_code") }}'
+          - name: user101
+            description: '{{ doc("shared_column_reason_code") }}'
```

Comment:
**[Question, non-blocking]** I assume this is intentional, but just to double-check: Are these three fields all indeed reason code fields?

Reply:
Yep! They are `reason_code_1`, `reason_code_2`, etc.